### ISSUE TITLE
a11y: Email management table

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6009]
Several accessibility improvements can be made to the email management table under `/manage/account`:

* The `<table>` itself should have a `<caption>`, as the first child node. This is partially mitigated by the surrounding `<section>` and `<h2>` above.
* The `email_verification_label` macro should add `aria-hidden=true` to each `<i class="fa fa-{{ icon }}">` and `<i class="fa fa-check">`
* The `<table>` should include `scope=col` or another appropriate scope for each `<th>` node
